src/visor/rpc.go

```diff
@@ -33,21 +34,23 @@ func (rpc RPC) GetBlockchainMetadata(v *Visor) *BlockchainMetadata {
 }
 
 // GetUnspent gets unspent
-func (rpc RPC) GetUnspent(v *Visor) coin.UnspentPool {
-	return v.Blockchain.GetUnspent().Clone()
+func (rpc RPC) GetUnspent(v *Visor) *blockdb.UnspentPool {
+	return v.Blockchain.Unspent
```

Comment:
We used to have unspent pool in memory and is not thread safe, we return copy to avoid caller from changing, which is not necessary actually. Now our unspent pool is in db, it's thread safe.